javascript/linked-list/__tests__/linked-list.test.js

```diff
@@ -1,10 +1,82 @@
 'use strict';
 
-// Require our linked list implementation
-const LinkedList = require('../index');
+const { LinkedList } = require('./linkedList.js');
```

Comment:
change the require part back to `../index` because that the name of the file that has the Linked lists methods and it located outside the test folder.